core/core.go

```diff
@@ -403,6 +403,11 @@ func (c *Core) GetHorizon() uint64 {
 // BlockChain methods //
 //--------------------//
 
+// WriteBlock write the block to the bodydb database
+func (c *Core) WriteBlock(block *types.Block) {
+	c.sl.hc.bc.WriteBlock(block)
```

Comment:
Is there a reason you didn't wrap this method like we've wrapped all other methods? e.g. `c.WriteBlock()` -> `sl.WriteBlock()` -> `hc.WriteBlock()` ...

idc really, but it seems inconsistent with the rest of the interfaces.

Reply:
Good catch, will wrap it to maintain consistency